lakefile.lean

```diff
@@ -12,7 +12,7 @@ lean_lib «lean4-metaprogramming-book» where
 require mdgen from git
   "https://github.com/Seasawher/mdgen" @ "v1.3.0"
 
-require std from git "https://github.com/leanprover/std4" @ "v4.7.0"
+require "leanprover-community" / "batteries" @ git "main"
```

Comment:
Shouldn't we use the corresponding tag -- i.e. https://github.com/leanprover-community/batteries/releases/tag/v4.15.0-rc1 -- so this doesn't break when Batteries updates its Lean version?

Reply:
Batteries updates will not break the code in this book until a `lake update` is manually performed.

This is not a problem, as any updates to Batteries should eventually be followed.